share/shwap/p2p/shrex/shrex_getter/shrex.go

```diff
@@ -306,6 +306,16 @@ func (sg *Getter) GetNamespaceData(
 	}
 }
 
+func (sg *Getter) GetRangeNamespaceData(
+	_ context.Context,
+	_ *header.ExtendedHeader,
+	_ libshare.Namespace,
+	_, _ shwap.SampleCoords,
+	_ bool,
+) (shwap.RangeNamespaceData, error) {
+	return shwap.RangeNamespaceData{}, errors.New("not supported")
+}
```

Comment:
Why not implement in shrex, but implement in bitswap? We planing to remove bitswap soon, so perhaps makes sense to do the opposite